internal/subnet_test.go

```
@@ -0,0 +1,47 @@
+}
+
+func TestPrefixSubnetEvidenceTitles(t *testing.T) {
+	evidences := []*proto.Evidence{
+		{Title: "Subnet should set required tags"},
+		{Title: ""},
+		nil,
+	}
+
+	PrefixEvidenceTitles(evidences, "subnet-123")
+
+	if got := evidences[0].Title; got != "subnet-123 | Subnet should set required tags" {
```

golangci-lint (2.12.2)

[error] avoid direct access to proto field evidences[0].Title, use evidences[0].GetTitle() instead

(protogetter)